content/docs/command-reference/stage/add.md

```diff
@@ -0,0 +1,432 @@
+# stage add
+
+Helper command to add or update <abbr>stages</abbr> in `dvc.yaml`. Requires a
+name and a command.
+
+## Synopsis
+
+```usage
+usage: dvc stage add [-h] [-q | -v] -n <name> [-d <path>] [-o <path>]
+                     [-O <path>] [-p [<path>:]<params_list>] [-m <path>]
+                     [-M <path>] [--plots <path>] [--plots-no-cache <path>]
+                     [-w <path>] [-f]
+                     [--outs-persist <path>] [--outs-persist-no-cache <path>]
+                     [--always-changed] [--external] [--desc <text>]
+                    command
```

Comment:
> syntax highlighting fails. If I add one space or remove it, it mysteriously works

The lines are too long anyway, please try:
```suggestion
usage: dvc stage add [-h] [-q | -v] -n <name> [-d <path>] [-o <path>]
                     [-O <path>] [-p [<path>:]<params_list>]
                     [-m <path>] [-M <path>] [--plots <path>]
                     [--plots-no-cache <path>] [-w <path>] [-f]
                     [--outs-persist <path>]
                     [--outs-persist-no-cache <path>]
                     [--always-changed] [--external] [--desc <text>]
                     command
```

Reply:
Yup, that works. Thanks.